corehq/apps/domain/auth.py

```diff
@@ -302,3 +306,23 @@ def get_identifier(self, request):
         username = self.extract_credentials(request)[0]
         domain = getattr(request, 'domain', '')
         return ApiIdentifier(username=username, domain=domain)
+
+    def extract_credentials(self, request):
+        """Extract username and key from request"""
+        # This overrides an existing tastypie method
+        try:
+            data = self.get_authorization_data(request)
+        except ValueError:
+            if self._allow_creds_in_data:
+                username = request.GET.get('username') or request.POST.get('username')
+                api_key = request.GET.get('api_key') or request.POST.get('api_key')
+                if username and api_key:
+                    metrics_counter('commcare.auth.credentials_in_data', tags={
+                        'domain': getattr(request, 'domain', None),
```

Comment:
could you also log if this was a GET or POST request - I think that's useful info to have.